javaslang/generator/Generator.scala

```diff
@@ -1558,6 +1569,15 @@ def generateTestClasses(): Unit = {
         """
       }
 
+      def genSimpleAliasTest(name: String, value: String): String = {
+        xs"""
+          @$test
+          public void should${name}ReturnNotNull() {
+              assertThat(${name.toLowerCase}($value)).isNotNull();
```

Comment:
This is not best implementation, I am really want only `uncapitalize`, but currently simple `toLowerCase` work to, but not in all cases.
I will add function `uncapitalize` today